### DISCUSSION TITLE
Datepicker JS import?

### DISCUSSION THREAD

**Discussion by ArtFlag** [ant-design/ant-design#29116] [Q&A]
To avoid issues with Ant's global style overriding my app's styles, I'm importing components individually.

I've imported the following styles:

```
import 'antd/lib/time-picker/style/index.css'
import 'antd/lib/date-picker/style/index.css'
import { DatePicker, Form, Input, Button, InputNumber } from 'antd'
```

and it renders fine but the JS to validate the date is missing (`OK` is disabled, `now` doesn't work, etc).
<img width="442" alt="Screenshot 2021-01-29 at 09 23 05" src="https://user-images.githubusercontent.com/25686502/106251070-d194f400-6214-11eb-9e8f-70b001bfe46b.png">

How can I also import the rest of the logic?

Thanks!

**Comment by Vinayvinayu** [comment]
What is the solution for this. Even for me date picker is loaded but it is completely read only..
